spec/coffee/showIn.spec.coffee

```diff
@@ -3,6 +3,11 @@ describe 'Marionette.Component', ->
     @setFixtures '<div id="main"></div>'
 
   describe 'when showing', ->
+    options = {
+      model: Backbone.Model,
```

Comment:
Let's instead use a child of `Backbone.Model` via `Backbone.Model.extend`. Also this should be an instance, not the class itself.